settings.gradle.kts

```diff
@@ -14,10 +14,159 @@
+
+include(":app-service-token", "hedera-node/hedera-token-service")
+
+include(":app-service-token-impl", "hedera-node/hedera-token-service-impl")
+
+include(":app-service-util", "hedera-node/hedera-util-service")
+
+include(":app-service-util-impl", "hedera-node/hedera-util-service-impl")
+
+include(":app-spi", "hedera-node/hedera-app-spi")
+
+include(":config", "hedera-node/hedera-config")
+
+include(":hapi", "hedera-node/hapi")
+
+include(":services-cli", "hedera-node/cli-clients")
+
+include(":test-clients", "hedera-node/test-clients")
+
+// Platform SDK projects
+include(":swirlds-platform", "platform-sdk")
+
+include(":swirlds", "platform-sdk/swirlds")
+
+include(":swirlds-base", "platform-sdk/swirlds-base")
+
+include(":swirlds-logging", "platform-sdk/swirlds-logging")
+
+include(":swirlds-common", "platform-sdk/swirlds-common")
+
+include(":swirlds-sign-tool", "platform-sdk/swirlds-sign-tool")
+
+include(":swirlds-config-api", "platform-sdk/swirlds-config-api")
+
+include(":swirlds-config-impl", "platform-sdk/swirlds-config-impl")
+
+include(":swirlds-config-benchmark", "platform-sdk/swirlds-config-benchmark")
+
+include(":swirlds-fchashmap", "platform-sdk/swirlds-fchashmap")
+
+include(":swirlds-fcqueue", "platform-sdk/swirlds-fcqueue")
+
+include(":swirlds-merkle", "platform-sdk/swirlds-merkle")
+
+include(":swirlds-merkledb", "platform-sdk/swirlds-jasperdb")
+
+include(":swirlds-virtualmap", "platform-sdk/swirlds-virtualmap")
+
+include(":swirlds-platform-gui", "platform-sdk/swirlds-platform-gui")
+
+include(":swirlds-platform-core", "platform-sdk/swirlds-platform-core")
+
+include(":swirlds-cli", "platform-sdk/swirlds-cli")
+
+include(":swirlds-benchmarks", "platform-sdk/swirlds-benchmarks")
+
+include(":swirlds-test-framework", "platform-sdk/swirlds-unit-tests/common/swirlds-test-framework")
+
+include(":swirlds-common-testing", "platform-sdk/swirlds-unit-tests/common/swirlds-common-test")
+
+include(":swirlds-platform-test", "platform-sdk/swirlds-unit-tests/core/swirlds-platform-test")
+
+include(":swirlds-merkle-test", "platform-sdk/swirlds-unit-tests/structures/swirlds-merkle-test")
+
+// Platform demo/test applications
+includeAllProjects("platform-sdk/platform-apps/demos")
+
+includeAllProjects("platform-sdk/platform-apps/tests")
+
+fun include(name: String, path: String) {
+    include(name)
+    project(name).projectDir = File(rootDir, path)
+}
+
+fun includeAllProjects(containingFolder: String) {
+    File(rootDir, containingFolder).listFiles()?.forEach { folder ->
+        if (File(folder, "build.gradle.kts").exists()) {
+            val name = ":${folder.name}"
+            include(name)
+            project(name).projectDir = folder
+        }
+    }
+}
+
+// The HAPI API version to use for Protobuf sources. This can be a tag or branch
+// name from the hedera-protobufs GIT repo.
+val hapiProtoVersion = "0.43.0-rc-SNAPSHOT"
+val hapiProtoBranchOrTag = "add-pbj-types-for-state"
```

Comment:
Should we have a tag on the PBJ repo for the release instead of this branch?

Reply:
This is outside the scope of this PR and is what already shipped in the `0.43.0` release. This should be addressed with the Services Release manager for `0.43`.